ui/src/common/components/DateRangeAggregation.jsx

```diff
@@ -0,0 +1,119 @@
+import React, { useCallback, useMemo } from 'react';
+import PropTypes from 'prop-types';
+import moment from 'moment';
+import { DatePicker, Row } from 'antd';
+import AggregationBox from './AggregationBox';
+import {
+  DATE_RANGE_FORMAT,
+  RANGE_AGGREGATION_SELECTION_SEPARATOR,
+} from '../constants';
+
+function DateRangeAggregation({ name, onChange, selections }) {
+  const splitSelections = useMemo(
+    () => selections && selections.split(RANGE_AGGREGATION_SELECTION_SEPARATOR),
+    [selections]
+  );
+
+  const startDate = useMemo(
```

Comment:
reduntant

Reply:
also redundant use of `useMemo`. It should be used when you do "heavy" calculation or reguler one that returns always a different reference (which then causes rerendering if you pass it as prop to any child).